databricks_cli/unity_catalog/table_cli.py

```diff
@@ -150,12 +150,12 @@ def delete_table_cli(api_client, full_name):
     UnityCatalogApi(api_client).delete_table(full_name)
 
 
-@click.group()
+@click.group(context_settings=CONTEXT_SETTINGS,
+             short_help=' ',
```

Comment:
Why can't this be empty?

Reply:
Looks like when the `short_help` is empty, `click` uses the long `help` and truncates it in this case:
```
$ databricks unity-catalog --help
Usage: databricks unity-catalog [OPTIONS] COMMAND [ARGS]...

  Utility to interact with Databricks Unity Catalog.

Options:
[..]
  storage-credentials
  tables               Note: To create or update tables, use SQL commands...
```
So using \` \` is me being tricky.  :)